tests/test_library.py

```diff
@@ -430,6 +430,13 @@ def test_library_MusicSection_recentlyAdded(music, artist):
     assert track in music.recentlyAddedTracks()
 
 
+@pytest.mark.authenticated
+def test_library_MusicSection_sonicAdventure(account_plexpass, music):
+    tracks = music.searchTracks()
+    adventure = music.sonicAdventure(tracks[0], tracks[-1].ratingKey)
```

Comment:
the first track needs to be using the `ratingKey` attribute here too.

`music.sonicAdventure(tracks[0].ratingKey, tracks[-1].ratingKey)`

Reply:
We are testing that `sonicAdventure` accepts both `Track` objects and `int` rating keys.